crates/control/src/behavior/dribble.rs

```diff
@@ -249,6 +245,29 @@ fn find_targets_to_kick_to(
         possible_kick_targets.extend(emergency_targets);
     }
 
+    // Create obstacles kick targets to filter with
```

Comment:
Move all these structural comments to function names?

Reply:
Done